src/index.js

```diff
@@ -1,19 +1,23 @@
 const express = require("express"),
   path = require("path"),
   app = express(),
+  http= require("http"),
+  hostname= "127.0.0.1",
   port = process.env.PORT || 3000;
+  
+
 
 app.get('/', (req, res) => {
   let userInfo = req.header("user-agent");
   res.send(`UserInfo: ${userInfo}`);
 });
 
-app.get('/receipts', (req, res) => {
-  let file = path.join(__dirname, "asset/receipt.pdf");
+app.get('/receipts', (_req, res) => {
+  let file = path.join(__dirname, "src/asset/receipt.pdf");
```

Comment:
No es recomendable hacer referencia src directamente. utilicemos __dirname

https://www.digitalocean.com/community/tutorials/nodejs-how-to-use__dirname